config/manager/manager.yaml

```diff
@@ -36,4 +41,8 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
+        volumeMounts:
+            - mountPath: /.prcconfig
```

Comment:
I'm not sure why we should use a `.prcconfig` in the root filesystem. Maybe `/etc/prcconfig`, or `/opt/prcconfig` or `~/.prcconfig`?

Reply:
Good point, we should not. Which one do you prefer? I'm not unhappy with `~`, but maybe `/etc/` is a better place for it?

Reply:
👍  for `/etc/`